managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/EntryCacheTest.java

```diff
@@ -20,19 +20,14 @@
 
 import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.anyLong;
-import static org.mockito.Mockito.doAnswer;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
might be good to make this consistent, either use "import *" or just disallow importing "*". allow mixing them together is not very good.

Reply:
This change was actually a mistake. The Checkstyle config explicitly disallows star imports. I'll remove it.